Swiftfin tvOS/Views/SettingsView/VideoPlayerSettingsView.swift

```diff
@@ -43,46 +43,54 @@ struct VideoPlayerSettingsView: View {
             .contentView {
 
                 Section {
+
                     ChevronButton(
-                        "Resume Offset",
+                        L10n.offset,
                         subtitle: resumeOffset.secondLabel
                     )
                     .onSelect {
                         isPresentingResumeOffsetStepper = true
                     }
+                } header: {
+                    L10n.resume.text
                 } footer: {
-                    Text("Resume content seconds before the recorded resume time")
+                    L10n.resumeOffsetDescription.text
                 }
 
                 Section {
```

Comment:
Since you're here, please use the other init for both header and footer.

Reply:
I believe this is the same item as https://github.com/jellyfin/Swiftfin/pull/1163#discussion_r1706031863 so I will base this change on that discussion!

Reply:
https://github.com/jellyfin/Swiftfin/pull/1163#discussion_r1706053161